src/code/audio_effects.c

```diff
@@ -243,7 +243,7 @@ f32 Audio_AdsrUpdate(AdsrState* adsr) {
             adsr->envIndex = 0;
             adsr->action.s.state = ADSR_STATE_LOOP;
             FALLTHROUGH;
-        retry:
+        retry:;
```

Comment:
Of course it's audio causing troubles again

Reply:
It's always the loudy neighbours being an impediment